### ISSUE TITLE
False positives regarding module member detection with relative import

### ISSUE THREAD

**Issue by gousaiyang** [netromdk/vermin#42]
**Describe the bug**
When a module (imported with relative import) in my own package has the same name as a standard library module, vermin may emit false positive detection results.

**To Reproduce**

```console
user@machine:/tmp/test/mypackage
$ ls
__init__.py  main.py  typing.py
user@machine:/tmp/test/mypackage
$ cat main.py
from .typing import Final
user@machine:/tmp/test/mypackage
$ cat typing.py
from typing_extensions import Final
user@machine:/tmp/test/mypackage
$ vermin -vv .
Detecting python files..
Analyzing 3 files using 8 processes..
~2, ~3       /tmp/test/mypackage/__init__.py
!2, 3.8      /tmp/test/mypackage/main.py
  'typing' requires !2, 3.5
  'typing.Final' requires !2, 3.8

~2, ~3       /tmp/test/mypackage/typing.py
Tip: You're using potentially backported modules: typing
If so, try using the following for better results: --backport typing

Minimum required versions: 3.8
Incompatible versions:     2
```

**Expected behavior**
Should not detect `from .typing import Final` as 3.8+.

**Environment:**
Vermin version 0.10.2

**Comment by netromdk** [comment]
Thanks for reporting the issue. Currently, it just looks for the names of the imports and their full package names. But perhaps if it's a relative import it should ignore if the file is on disk. Like you have "./typing.py" as `.typing`. Or maybe it's enough to ignore all imports starting with `.` and the like.

**Comment by netromdk** [comment]
Ah, so I can just ignore modules that aren't top-level imports. For all imports I can see a "level" in the AST where it is 0 for top-level packages and 1 for `.typing`, 2 for `..typing` and so on.

https://docs.python.org/3/reference/import.html#package-relative-imports

**Comment by netromdk** [comment]
I've released [v0.10.3](https://github.com/netromdk/vermin/releases/tag/v0.10.3). Thanks again.

**Comment by gousaiyang** [comment]
Thanks for fixing this!
